wire-library/wire-tests/src/commonTest/proto/java/samebasename/single_level.proto

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package single_level;
+package samebasename.single_level;
```

Comment:
btw two files with same name + same java_package will crash on protoc because protoc will want to generate outer classes for each and they'll have the same name (even though each file might have different messages.)

Reply:
there is another `single_level.proto` one directory below.

Reply:
Related to #1544